index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200&family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header id="sub-container">
+      <nav class="navbar">
+        <a href="#" class="menu-icon">
+          <img src="./images/my-logo/logo3.jpg" alt="logo" width="35"
+        /></a>
+        <div class="menu">
+          <img src="./images/menu/Icon-Menu.png" alt="menu icon" />
+        </div>
+      </nav>
+    </header>
+    <section class="main-container">
+      <div class="logo-intro">
+        <img
+          src="./images/my-logo/logo-1.png"
+          alt="logo"
+          width="170"
+          class="my-image"
+        />
+        <h1 class="greeting">
+          Nice to meet you, <br />
+          I'm John
+        </h1>
+      </div>
+      <div class="sub-main-container">
+        <div class="intro">
+          <h3 class="profession">A Web Developer</h3>
+          <p class="welcome">
+            I can help you build a product , feature or <br />
+            website Look through some of my work <br />
+            and experience! If you like what you see <br />
+            and have a project you need coded,
+          </p>
+        </div>
+        <div class="design"><img src="./images/design.png" alt="" /></div>
+        <div class="connect-text">Don't hestiate to contact me.</div>
+        <div class="connect-link">
+          <ul>
+            <li>
+              <a href="https://github.com/Outkarm"
+                ><img
+                  src="./images/github-logo/github.png"
```

Comment:
- [ ] Please as suggested by the previous reviewer try to stick to the figma design as much as possible . you got what it takes 💯 

Your design:
![image](https://user-images.githubusercontent.com/90524466/185432885-e2932954-2141-4a3f-81b2-c0d2b01024b5.png)

Figma design:
![image](https://user-images.githubusercontent.com/90524466/185433341-134059fd-97b0-4c41-94a4-b633437729cd.png)

Reply:
done